src/attack_flow_builder/src/assets/builder.config.publisher.ts

```diff
@@ -224,7 +224,16 @@ class AttackFlowPublisher extends DiagramPublisher {
                 case PropertyType.Enum:
                     if (prop instanceof EnumProperty && prop.isDefined()) {
                         let value = prop.toReferenceValue()!.toRawValue()!;
```

Comment:
To simplify this, just export the identifier of the selected enum (instead of trying to convert the enum's text to the identifier).

1. Replace `let value = prop.toReferenceValue()!.toRawValue()!;` with `let value = prop.toRawValue();`

2. In the `// case(BoolEnum)` use lowercase `true` and `false`.

3. In the `// case(String | List | Dictionary | null )` use `value` outright.

Reply:
In ListProperty.ts, value is set with `this.value.set(MD5(id), prop)`
https://github.com/center-for-threat-informed-defense/attack-flow/blob/main/src/attack_flow_builder/src/assets/scripts/BlockDiagram/Property/ListProperty.ts#L49C17-L49C17

To get around changing ListProperty.ts, I calculated value using `let value = prop.toReferenceValue()!.toRawValue()!;` since `let value = prop.toRawValue()!;` returns an MD5 hash. I'll push along the changes above, and changing how ListProperty.ts handles `let [id, value] of descriptor.value`, just wanted to clarify the resulting change to ListProperty.ts